### ISSUE TITLE
Windows 10, ShareX always fails to register standard shortcuts because of Onedrive

### ISSUE THREAD

**Issue by soohoon90** [ShareX/ShareX#1144]
OneDrive seems to have added a new feature that captures PrtScr to onedrive. However this feature is turned off by default.

ShareX on Windows 10 fails to register for standard short-cuts.
Is it possible for ShareX to 'share' the shortcuts with Onedrive?

**Comment by Jaex** [comment]
It is not possible, Windows not allows it.

**Comment by zikeji** [comment]
You could use a low level keyboard hook to do shortcuts. The negative of
that is it gets picked up as a virus because of the hook. Not saying you
couldn't work around it.

**Comment by Jaex** [comment]
I were using low level keyboard hook long years ago in ZScreen and it was tend to randomly unhook in some people which was not reproduceable by me therefore I was changed it to this most used hotkey system which never unhooks and works more stable. Also I checked most of screen capture softwares and nearly all of them was using this hotkey system. Using keyboard hook is not preferred method by majority of software. And I won't suggest using it too.
